json-schemas/interface-methods/records-filter.json

```diff
@@ -26,6 +26,38 @@
     "schema": {
       "type": "string"
     },
+    "tags": {
+      "type": "object",
+      "minProperties": 1,
+      "additionalProperties": {
+        "oneOf": [
+          {
+            "type": "string"
+          },
+          {
+            "type": "number"
+          },
+          {
+            "type": "boolean"
+          },
+          {
+            "type": "object",
+            "minProperties": 1,
+            "additionalProperties": false,
+            "properties": {
+              "prefix": {
```

Comment:
Consider renaming to `startsWith` which is probably more conventional.

Reply:
Would be good to rename other related type, methods etc. (e.g. `PrefixFilter` -> `StartsWithFilter`, `isPrefixFilter` -> `isStartsWithFilter` etc)

Reply:
Done